DTS-STN/Alpha-Site
# Added GraphQL and other peer dependencies

# Description

Currently on our dev site, if you navigate to `/projects`, you get redirected to `/500`. In Log Stream we are getting `Error: Cannot find module 'graphql'` since Apollo has an unmet peer dependency of `"graphql@^14.0.0 || ^15.0.0 || ^16.0.0"`.

This PR adds the GraphQL package and takes care of the other unmet peer dependencies that were showing up if you were to do `yarn install --check-files`.

This PR also addresses the currently open Snyk PRs:
#507 , #506 , #505 , #504 

## Test Instructions

1. Pull in branch
2. Type `yarn install`
3. Type `yarn dev`
4. Ensure app works as expected

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96&projectKey=SCL)
